media/inf/hdc.inf

```diff
@@ -329,15 +308,14 @@ HKR, , Installer32,    0, "syssetup.dll,HdcClassInstaller"
 %PCI\VEN_ffff&DEV_ffff&REV_ff.DeviceDesc%=PciIde_Inst, PCI\VEN_ffff&DEV_ffff&REV_ff
 
 ;Compatible adapters
-
-%PCI\CC_0101.DeviceDesc% = PciIde_Inst,PCI\CC_0101                             ; Device ID for generic Dual PCI IDE  (UNKNOWN_PCI_IDE)
-%PCI\CC_0104.DeviceDesc% = uniata_Inst,PCI\CC_0104                             ; Device ID for generic Dual PCI IDE  (UNKNOWN_PCI_IDE)
-%PCI\CC_0105.DeviceDesc% = uniata_Inst,PCI\CC_0105                             ; Device ID for generic Dual PCI IDE  (UNKNOWN_PCI_IDE)
-%PCI\CC_0106.DeviceDesc% = uniata_Inst,PCI\CC_0106                             ; Device ID for generic Dual PCI IDE  (UNKNOWN_PCI_IDE)
-%*PNP0600.PriDeviceDesc% = uniata_Inst, Primary_IDE_Channel
-%*PNP0600.SecDeviceDesc% = uniata_Inst, Secondary_IDE_Channel
-%*PNP0600.DeviceDesc%    = uniata_Inst, *PNP0600                                ; Generic_ESDI_Hard_Disk_Controller   (BAD_IDE)
-%UniATA_Name% = uniata_Inst
+%PCI\CC_0101.DeviceDesc%=PciIde_Inst,PCI\CC_0101                       ; Device ID for generic Dual PCI IDE  (UNKNOWN_PCI_IDE)
```

Comment:
Please revert the space surrounding the equals sign...

Reply:
I did this change, because almost all Key-value-pairs in this file don't space pad the equals sign. Check the entire file.